pages/legend.py

```diff
@@ -3,13 +3,17 @@
 from utils.code_and_show import example_app
 
 
-dash.register_page(__name__, description="Interactively change the legend position")
+dash.register_page(__name__, description="Interactively change the legend position. It has a top-bottom layout and a regular-callback")
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
 
+#### Dash Components in App:
+- [RadioItems](https://dash.plotly.com/dash-core-components/radioitems)
+
+
 #### Plotly Documentation:  
 
 - [How to configure and style the legend](https://plotly.com/python/legend/)
```

Comment:
@milanzmitrovic can you please delete the empty space (blank line) between Plotly documentation and the link below it. 
Also, can you please add the a second link to the scatter plot page under the Plotly Documentation, since this app uses a scatter plot:
[Scatter Plots](https://plotly.com/python/line-and-scatter/)